.github/workflows/ci.yml

```
@@ -69,21 +96,111 @@ jobs:
         run: MIX_ENV=dev mix dialyzer --plt
 
       - name: Run Dialyzer
-        run: MIX_ENV=dev mix dialyzer --format github
+        id: dialyzer
+        run: |
+          echo "## Dialyzer Results" >> $GITHUB_STEP_SUMMARY
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)